lib_forceload/data/gm4_forceload-1.0/functions/check_dimension.mcfunction

```diff
@@ -4,4 +4,6 @@
 advancement revoke @s only gm4_forceload-1.0:dimension_marking
 
 # spawns a dimension marker if one doesn't already exist in the dimension
-execute unless entity @e[type=minecraft:area_effect_cloud,tag=gm4_dimension,distance=0..] run function gm4_forceload-1.0:mark_dimension
+execute as @e[type=area_effect_cloud,tag=gm4_dimension] at @s run function gm4_forceload-1.0:update_marker
+execute store success score $initialized gm4_dimension if block 29999998 1 7133 birch_wall_sign
+execute if score $initialized gm4_dimension matches 0 run function gm4_forceload-1.0:mark_dimension
```

Comment:
Why can this not be done in 1 command?
```mcfunction
execute unless block 29999998 1 7133 birch_wall_sign run function gm4_forceload-1.0:mark_dimension
```

Reply:
They changed how `unless block` works if the position isn't loaded, it will not run any commands after that. This is why I have to check the success of the `if block`